CRADLE/correctbiasutils/__init__.py

```diff
@@ -78,32 +78,22 @@ def mergeCorrectedBedfilesToBW(meta, bwHeader, dataInfo, repInfo, signalBWName):
 
 	return signalBWName
 
-def divideGenome(regions, binSize=50000):
+def divideGenome(regions, bpBinSize=1, genomeBinSize=50000):
+
+	# Adjust the genome bin size to be a multiple of base pair bin Size
+	genomeBinSize -= genomeBinSize % bpBinSize
+
 	tasks = []
 	for region in regions:
 		regionChromo = region[0]
 		regionStart = int(region[1])
 		regionEnd = int(region[2])
-		regionLen = regionEnd - regionStart
-
-		if regionLen <= binSize:
-			tasks.append(region)
-		else:
-			numBin = float(regionLen) / binSize
-			if numBin > int(numBin):
-				numBin = int(numBin) + 1
-
-			numBin = int(numBin)
-			for i in range(numBin):
-				start = regionStart + i * binSize
-				end = regionStart + (i + 1) * binSize
-				if i == 0:
-					start = regionStart
-				if i == (numBin -1):
-					end = regionEnd
-
-				tasks.append([regionChromo, start, end])
 
+		binStart = regionStart
+		while binStart < regionEnd:
```

Comment:
@Young-Sook is the end "address" for the task inclusive or exclusive? Should this comparison be `<=` so an end base isn't accidentally skipped (it would create a new "bin" with only 1 base in it)